src/fortran/client.F90

```diff
@@ -1444,4 +1464,259 @@ function use_tensor_ensemble_prefix(self, use_prefix) result(code)
   code = use_tensor_ensemble_prefix_c(self%client_ptr, logical(use_prefix,kind=c_bool))
 end function use_tensor_ensemble_prefix
 
+!> Control whether aggregation lists are prefixed
+function use_list_ensemble_prefix(self, use_prefix) result(code)
+  class(client_type),   intent(in) :: self       !< An initialized SmartRedis client
+  logical,              intent(in) :: use_prefix !< The prefix setting
+  integer(kind=enum_kind)          :: code
+
+  code = use_list_ensemble_prefix_c(self%client_ptr, logical(use_prefix,kind=c_bool))
+end function use_list_ensemble_prefix
+
+!> Appends a dataset to the aggregation list When appending a dataset to an aggregation list, the list will
+!! automatically be created if it does not exist (i.e. this is the first entry in the list). Aggregation 
+!! lists work by referencing the dataset by storing its key, so appending a dataset to an aggregation list
+!! does not create a copy of the dataset.  Also, for this reason, the dataset must have been previously 
+!! placed into the database with a separate call to put_dataset().
+function append_to_list(self, list_name, dataset) result(code)
+  class(client_type), intent(in) :: self       !< An initialized SmartRedis client
+  character(len=*),   intent(in) :: list_name    !< Name of the dataset to get
+  type(dataset_type), intent(in) :: dataset !< receives the dataset
```

Comment:
Docstring labels on vars aren't lined up

Reply:
That appears to apply to many of the new procedures

Reply:
Fixed